### ISSUE TITLE
can you please organize your code?

### ISSUE THREAD

**Issue by bochen2014** [JonnyBurger/remotion-trailer#7]
something like 

frames
  ├── frame1
  ├── frame2
  ├── frame3
   ...

the code is very hard to follow atm

**Comment by JonnyBurger** [comment]
Hey Bo!

I have to say this structure would in my opinion be very inflexible, the fun of it is that is not linear, but that you can move things around, rearrange, and compose them, right?

Sorry for accidentially clicking close, but I welcome your suggestions for how to make it cleaner.
